t3f/kronecker.py

```diff
@@ -1,7 +1,7 @@
 import numpy as np
 import tensorflow as tf
 
-from t3f.tensor_train import TensorTrain
+from t3f import TensorTrain, TensorTrainBatch
```

Comment:
Please, use 
from t3f.tensor_train import TensorTrain
from t3f.tensor_train import TensorTrainBatch

I have no idea if there is any real difference, but 1) style guides usually suggest to not to import several things in one line 2) TF uses full absolute paths in imports (e.g. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/array_ops.py)